### PR TITLE
chore: bump II and NNS dapp versions

### PR DESCRIPTION
**Pull request by mraszyk**
This PR bumps II and NNS dapp to their latest mainnet verions:
- II: release-2024-05-13
- NNS-Dapp: proposal-129748

and installs their dependencies (ICRC1 ledger and index for ckETH, ICP index, SNS aggregator).

This PR has been tested manually: II login from NNS dapp, canister creation from NNS dapp, ICP transfer and transfer listing, and neuron staking from NNS dapp all work. It is expected to see the warning "There was an unexpected error while loading the summaries of all deployed projects. Error converting data from aggregator canister" until the first NNS proposal to create an SNS is executed.

### CODE REVIEW

**Review by ericswanson-dfinity — APPROVED** [review]

File: extensions/nns/src/install_nns.rs [hunk 1]
```diff
@@ -100,12 +169,50 @@ pub async fn install_nns(
...
+            let nns_dapp_init_args = Some(CanisterArguments {
+                args: nns_dapp_metadata,
+                schema: Some(SchemaLabel::AccountsInStableMemory),
+            });
+            Some(Encode!(&nns_dapp_init_args).unwrap())
+        } else if *canister_id == SNS_AGGREGATOR.canister_id {
+            Some(
+                Encode!(&Some(Config {
+                    update_interval_ms: 1_000,
+                    fast_interval_ms: 100,
```
Comment (dskloetd) [inline extensions/nns/src/install_nns.rs:202]: These parameters are crazy fast and cause a lot of unnecessary updates and a lot of spam in the local replica logs.

We typically use
```
update_interval_ms: 120_000,
fast_interval_ms: 10_000,
```
and even that produces a lot of output.

I've already made a change to stop producing all this output (which will only take effect after the next release) but even then it's unnecessary to make the aggregator do so much work.

Comment (mraszyk) [reply extensions/nns/src/install_nns.rs:202]: With a long update interval, you need to wait for 2 mins until your SNS appears on the dashboard, no?

Comment (dskloetd) [reply extensions/nns/src/install_nns.rs:202]: In snsdemo we temporarily change the configuration until all the SNSes are loaded:
https://github.com/dfinity/snsdemo/blob/main/bin/dfx-sns-aggregator-wait

I'm working around these parameters in snsdemo by reinstalling the aggregator so this doesn't need to be fixed for us.
But it definitely doesn't seem ideal like this.

Comment (mraszyk) [reply extensions/nns/src/install_nns.rs:202]: > But it definitely doesn't seem ideal like this.

Why? To reduce the amount of logs? If so, then ideally the log level is configurable in the canister init args and the nns extension of dfx could set an appropriate log level.

> In snsdemo we temporarily change the configuration

What's the alternative for the dfx extension? Creating a dfx extension command to reconfigure the aggregator and tell people to run it before and after their SNS deployment?

**Pull request merged**